packages/components/src/components/SquiggleChart.tsx

```diff
@@ -49,43 +45,35 @@ export const SquiggleChart: React.FC<SquiggleChartProps> = ({
   width = 500,
   height = 60,
 }: SquiggleChartProps) => {
-  let samplingInputs: SamplingInputs = {
+  let samplingInputs: samplingParams = {
     sampleCount: sampleCount,
-    outputXYPoints: outputXYPoints,
-    kernelWidth: kernelWidth,
-    pointDistLength: pointDistLength,
+    xyPointLength: outputXYPoints
   };
 
   let result = run(squiggleString, samplingInputs, environment);
```

Comment:
can we call this `squiggleResult`?

Reply:
Bit vague, I think `expressionResult` might work a bit better. Renamed to that